app/views/dashboard/broadcasts/_broadcast.html.haml

```diff
@@ -4,6 +4,20 @@
   %td
     = broadcast.channel.humanize
   %td
-    = broadcast.status.humanize
+    - if broadcast.running?
+      .span.badge.bg-blue.text-blue-fg
+        %i.icon.ti.ti-player-play
+        = broadcast.status.humanize
+    - elsif broadcast.stopped?
```

Comment:
Because we’re going to need to translate this into Khmer and Lao we could use enumerize here:
https://github.com/brainspec/enumerize?tab=readme-ov-file#i18n-support

Reply:
Exactly, we will do that. At this stage, I only focus on playing some ideas back and forth on UI how it going to look, and didn't pay attention on refactoring until we get what we want first.